lib/internal/bootstrap_node.js

```diff
@@ -206,23 +206,34 @@
     global.process = process;
     const util = NativeModule.require('util');
 
-    // Deprecate GLOBAL and root
-    ['GLOBAL', 'root'].forEach(function(name) {
-      // getter
-      const get = util.deprecate(function() {
+    function makeGetter(name) {
```

Comment:
Perhaps these could be named a bit clearer, like `makeDeprecateGetter()` or maybe `makeDepGetter()`, and similarly with `makeSetter()`.